src/tiles/list/tile-list-compact.ts

```diff
@@ -60,6 +62,14 @@ export class TileListCompact extends LitElement {
     `;
   }
 
+  private get href(): string {
+    // Use the server-specified href if available.
+    // Otherwise, construct a details page URL from the item identifier.
+    return this.model?.href
+      ? `${this.baseNavigationUrl}${this.model?.href}`
+      : `${this.baseNavigationUrl}/details/${this.model?.identifier}`;
```

Comment:
QUESTION: Is there a way to test both of these code pathways?

Reply:
Currently, full-text searches receive an `href` from the PPS, while metadata searches do not. So full-text should follow the top branch, while metadata should follow the bottom branch.

In practice, the only difference between the two is that FTS links append the current search query so that the search inside pane in BookReader opens automatically when you visit the item. However, in the future, TV and Radio searches will also provide custom `href` fields with things like start/end times set.